src/flowMC/resource_strategy_bundle/RQSpline_HMC.py

```
@@ -43,20 +45,31 @@ def __init__(
         n_training_loops: int,
         n_production_loops: int,
         n_epochs: int,
+        # --- Local sampler ---
         hmc_step_size: float = 0.1,
         hmc_n_leapfrog: int = 10,
         condition_matrix: Float | Float[Array, " n_dim"] = 1,
```

Ruff (0.15.2)

[error] Syntax error in forward annotation: Expected an expression

(F722)